sdk/maps/maps-common/review/maps-common.api.md

```diff
@@ -0,0 +1,187 @@
+## API Report File for "@azure/maps-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { CancelOnProgress } from '@azure/core-lro';
+import { KeyCredential } from '@azure/core-auth';
+import { PipelinePolicy } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+
+// @public
+export interface BatchPoller<TBatchResult> extends PollerLike<PollOperationState<TBatchResult>, TBatchResult> {
+}
+
+// @public
+export class BatchPollerProxy<TBatchResult, TInternalBatchResult> implements BatchPoller<TBatchResult> {
+    constructor(internalPoller: PollerLike<PollOperationState<TInternalBatchResult>, TInternalBatchResult>, mapper: (res: TInternalBatchResult) => TBatchResult);
+    // (undocumented)
+    cancelOperation(options?: {
+        abortSignal?: AbortSignalLike;
+    }): Promise<void>;
+    // (undocumented)
+    getOperationState(): PollOperationState<TBatchResult>;
+    // (undocumented)
+    getResult(): TBatchResult | undefined;
+    // (undocumented)
+    isDone(): boolean;
+    // (undocumented)
+    isStopped(): boolean;
+    // (undocumented)
+    onProgress(callback: (state: PollOperationState<TBatchResult>) => void): CancelOnProgress;
+    // (undocumented)
+    poll(options?: {
+        abortSignal?: AbortSignalLike;
+    }): Promise<void>;
+    // (undocumented)
+    pollUntilDone(): Promise<TBatchResult>;
+    // (undocumented)
+    stopPolling(): void;
+    // (undocumented)
+    toString(): string;
+}
+
+// @public
+export type BBox = BBox2D | BBox3D;
+
+// @public
+export type BBox2D = [
+southWestLongitude: number,
+southWestLatitude: number,
+northEastLongitude: number,
+northEastLatitude: number
+];
+
+// @public
+export type BBox3D = [
+southWestLongitude: number,
+southWestLatitude: number,
+southWestElevation: number,
+northEastLongitude: number,
+northEastLatitude: number,
+northEastElevation: number
+];
+
+// @public
+export interface BoundingBox {
+    bottomRight: LatLon;
+    topLeft: LatLon;
+}
+
+// @public
+export function createAzureMapsKeyCredentialPolicy(azureKeyCredential: KeyCredential): PipelinePolicy;
+
+// @public
+export function createMapsClientIdPolicy(mapsClientId: string): PipelinePolicy;
+
+// @public
+export interface GeoJsonFeature extends GeoJsonObject {
+    // (undocumented)
+    geometry?: GeoJsonGeometry;
+    // (undocumented)
+    id?: number | string;
+    // (undocumented)
+    properties?: Record<string, any>;
+    // (undocumented)
+    type: "Feature";
```

Comment:
would it be possible to name these properties `kind` instead of type? It is more commonly used as a discriminator in our libraries

Reply:
This is the [GeoJSON spec](https://www.rfc-editor.org/rfc/rfc7946#section-3.2:~:text=A%20Feature%20object%20has%20a%20%22type%22%20member%20with%20the%20value%20%22Feature%22.), so I think it would be better to keep the convention